spec/lib/snowplow_ruby_duid/domain_userid.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Creating a domain userid
+
+  Background:
+    Given a request environment
+      And the time is '2015-01-22 15:26:31 +0000'
+
+  Scenario: The domain userid is 16 characters long
+    When I create a domain userid
+    Then the domain userid has a length of '16'
+
+  Scenario Outline: The time is a component of the domain userid fingerprint
```

Comment:
Yes, it is. See: https://github.com/snowplow/snowplow-javascript-tracker/blob/d3d10067127eb5c95d0054c8ae60f3bdccba619d/src/js/tracker.js#L468-L472